src/components/ContentSidebar/ActivityFeed/task/Task.js

```diff
@@ -130,6 +129,8 @@ class Task extends React.Component<Props> {
                                                     }
                                                 />
                                             );
+                                        default:
```

Comment:
Should we avoid rendering the whole task if there is no assignee? We would have to bump the check up a level.

Reply:
I can make sure we check for that.